pylibs/stress_tests/128_children.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022, The OTNS Authors.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+# 1. Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+# 2. Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+# 3. Neither the name of the copyright holder nor the
+#    names of its contributors may be used to endorse or promote products
+#    derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+#
+import logging
+import math
+import random
+
+from BaseStressTest import BaseStressTest
+
+PARENT_X = 500
+PARENT_Y = 500
+MAX_DISTANCE = 400
+RADIO_RANGE = int(MAX_DISTANCE * 1.2)
+CHILDREN_N = 128
+
+
+class StressTest(BaseStressTest):
+    SUITE = 'network-limits'
+
+    TIME_LIMIT = {
+        'fed': 3,
+        'med': 3,
+        'sed': 3,
+    }
+
+    def __init__(self):
+        super(StressTest, self).__init__("Parent with 128 Children",
+                                         [])
+
+    def run(self):
+        self.test('fed')
+        self.test('med')
+        self.test('sed')
+
+    def test(self, child_type: str):
+        self.reset()
+        parent = self.ns.add("router", PARENT_Y, PARENT_Y, radio_range=RADIO_RANGE)
```

Comment:
Not sure what is X and Y. Should this be `PARENT_X` here? (Thought X equals to Y here).